tools/mirror-pypi/mirror_pytorch_triton_rocm.sh

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env bash
+
+############################################################################
+#
+# Upload versions of pytorch-triton-rocm to download.pytorch.org
+#
+# Usage:
+#     bash mirror_pytorch_triton_rocm.sh
+#
+############################################################################
+
+set -eou pipefail
+
+VERSION=${VERSION:-2.0.0.dev20230218}
+TMPDIR=$(mktemp -d)
+
+trap 'rm -rf ${TMPDIR};' EXIT
+
+(
+    pushd "${TMPDIR}" >/dev/null
+    for abi in 37m 38 39 310 311; do
```

Comment:
Is 37m required? Or is it a courtesy support? We are not release 3.7 binaries so the download count might be very few.

Reply:
They had it uploaded so I just mirrored it, we probably don't need it but no harm in uploading it either.